.isort.cfg

```diff
@@ -0,0 +1,5 @@
+[settings]
+line_length = 100
+multi_line_output = 3
+include_trailing_comma = True
+known_third_party = alembic,click,dataclasses,dateutil,flask,flask_migrate,flask_restful,flask_sqlalchemy,mixer,pytest,slack,slackeventsapi,sqlalchemy
```

Comment:
Do we have to fill this in manually? What does it mean to be on this list?

Reply:
This is filled in by `seed-isort-config` which gets run in `pre-commit` before `isort`. The third party group of imports usually comes after the builtin standard Python library imports (like `datetime`), but before all of our `nb2` imports. This list just explicitly declares them and I think it makes it easier for `isort` to put them in the right group. We shouldn't have to touch this list though. It should get auto updated on commits when we add new third parties.